### PR TITLE
FAQ for Azure Quantum Credits

### PR DESCRIPTION
**Pull request by Mobius5150**
Adding FAQ and resources for Azure Quantum Credits users.

### CODE REVIEW

File: articles/credits-faq.md [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+## I'm encountering a problem using my credits, what can I do?
+If you are having any problems with Azure Quantum pertaining to credits or otherwise, the best and fastest place to get support is by [filing a support ticket with Azure](https://azure.microsoft.com/support/create-ticket/).
+
+If you do not have an Azure support subscription or are unsure about how to file a ticket, you may reply to your approval notice email and one of our team members will get back to you.
+
+## Guide: Onboarding with Honeywell Credits
+Instructions for configuring your Azure Quantum Workspace to access Honeywell:
+
+1. [Open the Azure Quantum Workspace Create screen in the Azure Portal](https://portal.azure.com/#create/Microsoft.AzureQuantum)
+1. On the **Basics** tab while configuring your Workspace, ensure the selected **subscription** matches the subscription you supplied in your onboarding information. The subscription information is also copied in the onboarding mail you received.
```
Comment (shenyingjun) [inline articles/credits-faq.md:78]: dropdown will show subscription name, but in form we are asking subscription id. Can we add a step in between to show them how to find the the name?

Comment (Mobius5150) [reply articles/credits-faq.md:83]: Good idea, I've added this in the latest rev.

File: articles/credits-faq.md [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+
+## I have applied, how long will it take to hear back on my application?
+We try to respond to all credits applications within one (1) month of the application date.
+
+## What programming languages or frameworks can be used with Azure Quantum and Azure Quantum Credits?
+Credits may be used with any programming language or framework that is supported by Azure Quantum. Currently we support:
+
+- Q# ([Quickstart](xref:microsoft.quantum.quickstarts.computing)) ([Installation](https://docs.microsoft.com/azure/quantum/quickstart-microsoft-qc))
+- Cirq ([Quickstart](xref:microsoft.quantum.quickstarts.computing.cirq)) ([Installation](https://quantumai.google/cirq/azure-quantum/access))
+- Qiskit ([Quickstart](xref:microsoft.quantum.quickstarts.computing.qiskit)) ([Installation]())
```
Comment (Bradben) [inline articles/credits-faq.md:39]: This is confusing - 

- For Q#, the quickstart link and the install link are the same.
- For Cirq, the Google install page just has our quickstart instructions repeated. 
- Not sure about Qiskit installation...

Can we just link to our quickstarts?

File: articles/credits-faq.md [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+
+## My application was rejected, can I appeal the decision or re-apply?
+Rejections may not be appealed, and may be made for a number of reasons including the contents of the application, or other business reasons. In the event of substantial changes to your proposed project or the group carrying out the project you may re-apply.
```
Comment (Bradben) [inline articles/credits-faq.md:42]:
```suggestion
## My application was rejected. Can I appeal the decision or re-apply?
Rejections may be made for a number of reasons, including the contents of the application or other business reasons, and may not be appealed. In the event of substantial changes to your proposed project, or the group carrying out the project, you may re-apply.
```

File: articles/credits-faq.md [hunk 4]
```diff
@@ -0,0 +1,107 @@
...
+
+In some cases, your rejection notice may specify that insufficient information was provided to evaluate the application, or note that resources that were linked to in the application were not accessible to reviewers. In this case, you may re-apply to the program while providing more information.
+
+# FAQ: Onboarding and using your credits
+
+## I received my approval notice, what happens next?
+Once you receive your approval notice, you will need to open the onboarding survey linked from the notice and fill out all of the onboarding information. Once that's complete your credits will be made available either within two (2) weeks, or on the start date of your credits grant (whichever is later).
+
+You will receive a notice once your credits are available to be used, with detailed instruction to do so.
+
```
Comment (Bradben) [inline articles/credits-faq.md:52]:
```suggestion
## I received my approval notice. What happens next?
Your approval notice will contain a link to an onboarding survey which you will need to fill out completely. Once you have completed the survey, your credits will be made available within two (2) weeks or on the start date of your credits grant, whichever is later.

You will receive a notice once your credits are available to be used, with detailed instruction to do so.

```

File: articles/credits-faq.md [hunk 5]
```diff
@@ -0,0 +1,107 @@
...
+## Are there any costs or charges when using my credits?
+There are no costs or charges to using your free credits, however there may be some small storage costs as the input and output of your credits jobs are stored in a storage account that you pay for. Job data is typically <1MB per job, and [blob storage pricing is detailed on this page](https://azure.microsoft.com/pricing/details/storage/blobs/).
+
```
Comment (Bradben) [inline articles/credits-faq.md:55]:
```suggestion
There are no costs or charges to using your free credits. However, there may be some small storage costs, as the input and output of your credits jobs are stored in a storage account that you pay for. Job data is typically <1MB per job. For more details, see [Azure Blob Storage pricing](https://azure.microsoft.com/pricing/details/storage/blobs/).
```

File: articles/credits-faq.md [hunk 6]
```diff
@@ -0,0 +1,107 @@
...
+## How do I see my credits balance?
+You can see your current credit balance by going to the Overview blade on your Azure Quantum Workspace, and selecting the "Quota" tab. You will see a credits quota for each provider that you have credits with. You will see the total number of credits you have received, and may hover over the relevant graph to see the amount you have used towards your credits.
```
Comment (Bradben) [inline articles/credits-faq.md:57]:
```suggestion
## How do I see my credits balance?
You can see your current credit balance by selecting the **Overview** blade of your Azure Quantum Workspace, and selecting the **Quota** tab. You will see a credits quota for each provider that you have credits with. The quota lists the total number of credits you have received, and you can hover over the relevant graph to see the amount you have used towards your credits.
```

File: articles/credits-faq.md [hunk 7]
```diff
@@ -0,0 +1,107 @@
...
+
+You can also see your quotas by using the AZ CLI and the `az quantum workspace quotas` command, like in the example below. In this case, the `qgs` row shows that the account has a total of `8333334 qgs` with IonQ, of which `33334 qgs` have been used. The account also has `800 HQC` with Honeywell, of which `0 HQC` have been used.
```
Comment (Bradben) [inline articles/credits-faq.md:59]:
```suggestion
You can also see your quotas by using the Azure CLI and the `az quantum workspace quotas` command, as shown in the following example. In this case, the `qgs` row shows that the account has a total of `8333334 qgs` with IonQ, of which `33334 qgs` have been used. The account also has `800 HQC` with Honeywell, of which `0 HQC` have been used.
```

File: articles/credits-faq.md [hunk 8]
```diff
@@ -0,0 +1,107 @@
...
+```bash
+$ az quantum workspace quotas -o table
+Dimension  Holds  Limit      Period    ProviderId  Scope         Utilization
+---------  -----  ---------  --------  ----------  ------------  -----------
+qgs        0.0    8333334.0  Infinite  ionq        Subscription  33334.0
+hqc        0.0    800.0      Infinite  honeywell   Subscription  0.0
+```
+
+Note that credits are shown in the quota units used by the provider, and not in the dollar value of the credits received. You may use the table below to understand the approximate conversion value between quota units and US dollar value:
```
Comment (Bradben) [inline articles/credits-faq.md:68]:
```suggestion
Note that credits are shown in the quota units used by the provider, and not in the dollar value of the credits received. Use the table below to understand the approximate conversion value between quota units and US dollar value:
```

File: articles/credits-faq.md [hunk 9]
```diff
@@ -0,0 +1,107 @@
...
+| Provider | Unit | Approximate $ Value (USD) | Documentation |
+|----------|------|---------------------------|---------------|
+| Honeywell | HQC (Honeywell Quantum Credit) | $1 USD ~= 0.08 HQC | [Learn more about how Honeywell Quantum Credits are calculated](xref:microsoft.quantum.providers.honeywell) |
+| IonQ | QGS (Qubit Gate-Shot) | $1 USD ~= 33,333.3 QGS | [Learn more about how IonQ Qubit-Gate-Shots are calculated](xref:microsoft.quantum.providers.ionq) |
+
+## I'm encountering a problem using my credits, what can I do?
+If you are having any problems with Azure Quantum pertaining to credits or otherwise, the best and fastest place to get support is by [filing a support ticket with Azure](https://azure.microsoft.com/support/create-ticket/).
```
Comment (Bradben) [inline articles/credits-faq.md:75]:
```suggestion
## I'm encountering a problem using my credits. What can I do?
If you are having any problems with Azure Quantum pertaining to credits, or anything else, the best and quickest way to get support is to [file a support ticket with Azure](https://azure.microsoft.com/support/create-ticket/).
```

File: articles/credits-faq.md [hunk 10]
```diff
@@ -0,0 +1,107 @@
...
+
+If you do not have an Azure support subscription or are unsure about how to file a ticket, you may reply to your approval notice email and one of our team members will get back to you.
+
+## Guide: Onboarding with Honeywell Credits
+Instructions for configuring your Azure Quantum Workspace to access Honeywell:
+
+1. [Open the Azure Quantum Workspace Create screen in the Azure Portal](https://portal.azure.com/#create/Microsoft.AzureQuantum)
```
Comment (Bradben) [inline articles/credits-faq.md:82]:
```suggestion
1. Open the [Azure Quantum Workspace Create page] in the Azure Portal(https://portal.azure.com/#create/Microsoft.AzureQuantum)
```

File: articles/credits-faq.md [hunk 11]
```diff
@@ -0,0 +1,107 @@
...
+1. On the **Basics** tab while configuring your Workspace, ensure the selected **subscription** matches the subscription you supplied in your onboarding information. The subscription information is also copied in the onboarding mail you received.
+    > Not sure how to find the subscription with the ID specified in the email? You can see the list of your [Azure Subscriptions on this page](https://portal.azure.com/#blade/Microsoft_Azure_Billing/SubscriptionsBlade) to identify which should be selected.
```
Comment (Bradben) [inline articles/credits-faq.md:84]:
```suggestion
> [!TIP]
> Not sure how to find the subscription with the ID specified in the email? View the [list of your Azure Subscriptions](https://portal.azure.com/#blade/Microsoft_Azure_Billing/SubscriptionsBlade) to identify which should be selected.
```

File: articles/credits-faq.md [hunk 12]
```diff
@@ -0,0 +1,107 @@
...
+1. On the **Basics** tab while configuring your Workspace, ensure the selected **region** matches the subscription you supplied in your onboarding information. The region information is also copied in the onboarding mail you received.
+1. Provider and SKU: On the **Providers** tab, select "Honeywell Quantum Solutions". When the SKU/Pricing selector opens, ensure you select "Azure Quantum Credits".
```
Comment (Bradben) [inline articles/credits-faq.md:86]:
```suggestion
1. On the **Providers** tab, select **Honeywell Quantum Solutions**. When the SKU/Pricing selector opens, ensure you select **Azure Quantum Credits**.
```

File: articles/credits-faq.md [hunk 13]
```diff
@@ -0,0 +1,107 @@
...
+    > Important: If you select any plan other than “Azure Quantum Credits” you may be charged for usage. If you do not see the Azure Quantum Credits SKU please double check that you have selected the correct Subscription and Region, then send an email to the support email address specified in your credits onboarding email if you cannot see the SKU.
```
Comment (Bradben) [inline articles/credits-faq.md:87]:
```suggestion
> [!IMPORTANT] 
> If you select any plan other than **Azure Quantum Credits**, you may be charged for usage. If you do not see the Azure Quantum Credits SKU, please double check that you have selected the correct **Subscription** and **Region**, then send an email to the support email address specified in your credits onboarding email if you cannot see the SKU.
```

File: articles/credits-faq.md [hunk 14]
```diff
@@ -0,0 +1,107 @@
...
+
+> If you have received grants towards more than one Provider, you may add all of them to the same Quantum Workspace or you may create multiple Workspaces.
```
Comment (Bradben) [inline articles/credits-faq.md:89]:
```suggestion
> [!TIP]
 > If you have received grants towards more than one Provider, you may add all of them to the same Quantum Workspace or you may create multiple Workspaces.
```

File: articles/credits-faq.md [hunk 15]
```diff
@@ -0,0 +1,107 @@
...
+Links to Quickstart guides (follow the one for your desired programming framework):
+
+## Guide: Onboarding with IonQ Credits
+
+1. [Open the Azure Quantum Workspace Create screen in the Azure Portal](https://portal.azure.com/#create/Microsoft.AzureQuantum)
```
Comment (Bradben) [inline articles/credits-faq.md:94]:
```suggestion
1. Open the [Azure Quantum Workspace Create page] in the Azure Portal(https://portal.azure.com/#create/Microsoft.AzureQuantum)
```

File: articles/credits-faq.md [hunk 16]
```diff
@@ -0,0 +1,107 @@
...
+1. On the **Basics** tab while configuring your Workspace, ensure the selected **subscription** matches the subscription you supplied in your onboarding information. The subscription information is also copied in the onboarding mail you received.
+    > Not sure how to find the subscription with the ID specified in the email? You can see the list of your [Azure Subscriptions on this page](https://portal.azure.com/#blade/Microsoft_Azure_Billing/SubscriptionsBlade) to identify which should be selected.
```
Comment (Bradben) [inline articles/credits-faq.md:96]:
```suggestion
 > [!TIP]
> Not sure how to find the subscription with the ID specified in the email? View the [list of your Azure Subscriptions](https://portal.azure.com/#blade/Microsoft_Azure_Billing/SubscriptionsBlade) to identify which should be selected.
```

File: articles/credits-faq.md [hunk 17]
```diff
@@ -0,0 +1,107 @@
...
+1. On the **Basics** tab while configuring your Workspace, ensure the selected **region** matches the subscription you supplied in your onboarding information. The region information is also copied in the onboarding mail you received.
+1. Provider and SKU: On the **Providers** tab, select "IonQ". When the SKU/Pricing selector opens, ensure you select "Azure Quantum Credits".
+    > Important: If you select any plan other than “Azure Quantum Credits” you may be charged for usage. If you do not see the Azure Quantum Credits SKU please double check that you have selected the correct Subscription and Region, then send an email to the support email address specified in your credits onboarding email if you cannot see the SKU.
```
Comment (Bradben) [inline articles/credits-faq.md:99]:
```suggestion
> [!IMPORTANT] 
> If you select any plan other than **Azure Quantum Credits**, you may be charged for usage. If you do not see the Azure Quantum Credits SKU, please double check that you have selected the correct **Subscription** and **Region**, then send an email to the support email address specified in your credits onboarding email if you cannot see the SKU.       
    
```

File: articles/credits-faq.md [hunk 18]
```diff
@@ -0,0 +1,107 @@
...
+
+> If you have received grants towards more than one Provider, you may add all of them to the same Quantum Workspace or you may create multiple Workspaces.
```
Comment (Bradben) [inline articles/credits-faq.md:101]:
```suggestion
> [!TIP]
 > If you have received grants towards more than one Provider, you may add all of them to the same Quantum Workspace or you may create multiple Workspaces.
```

File: articles/credits-faq.md [hunk 19]
```diff
@@ -0,0 +1,107 @@
...
+Links to Quickstart guides (follow the one for your desired programming framework):
+
+## I Couldn't find an answer to my question, how can I get support?
+If you have an Azure support subscription, we recommend connecting with Azure Support for the fastest resolution to your question.
```
Comment (Bradben) [inline articles/credits-faq.md:105]:
```suggestion
If you have an Azure support subscription, we recommend connecting with [Azure Support](https://azure.microsoft.com/support/options/#get-support) for the fastest resolution to your question.
```

File: articles/credits-faq.md [hunk 20]
```diff
@@ -0,0 +1,107 @@
...
+
+If you have any question regarding the workspace creation process or run into any issue at any point using Azure Quantum, bookmark [https://aka.ms/AQ/OfficeHours](https://aka.ms/AQ/OfficeHours) and join our open office hours every Thursday at 8:30am PST.
```
Comment (Bradben) [inline articles/credits-faq.md:107]:
```suggestion
If you have any questions regarding the workspace creation process, or any issue using Azure Quantum, bookmark [https://aka.ms/AQ/OfficeHours](https://aka.ms/AQ/OfficeHours) and join our open office hours every Thursday at 8:30am PST.
```

**Review by Bradben — COMMENTED** [review]
Minor copy edits - thanks!

File: articles/credits-faq.md (hunk 2 above)
Comment (Mobius5150) [reply articles/credits-faq.md:39]: Yep, I think I'll just link the quickstarts

**Review by Bradben — APPROVED** [review]
Thanks!

**Pull request merged**